schemas/rum/error-schema.json

```diff
@@ -307,6 +307,12 @@
                 }
               },
               "readOnly": true
+            },
+            "disposition": {
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏Since this field will only be filled in the context of CSP errors, I wonder if it could be valuable to reflect it in the attribute path, either by prefixing it (`error.csp_disposition`) or nesting it (`error.csp.disposition`).
wdyt?
cc @amortemousque who reported the task